parmed/openmm/topsystem.py

```diff
@@ -230,7 +227,7 @@ def _process_angle(struct, force):
 def _process_urey_bradley(struct, force):
     """ Adds Urey-Bradley parameters to the structure """
     if not struct.angles:
-        warnings.warn('Adding what seems to be Urey-Bradley terms before '
+        warnings.warn('Adding what seems to be Urey-Bradley terms before ' # pragma: no cover
```

Comment:
No, I actually try to trigger all warnings (and most exceptions) in my tests.  This particular one is not important to trigger, though, so I decided to omit it rather than invent a test for it.